.github/workflows/sentry_sidekiq_test.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           - { ruby_version: 2.6, sidekiq_version: 5.0 }
           - { ruby_version: 2.6, sidekiq_version: 6.0 }
           - { ruby_version: jruby, sidekiq_version: 5.0 }
-          - { ruby_version: jruby, sidekiq_version: 6.0 }
+          - { ruby_version: jruby, sidekiq_version: 6.5 }
```

Comment:
With latest rubygems under jruby, bundling sidekiq using 6.0 as the version spec would install sidekiq 6.0.x - and that version actually crashes with `Sidekiq[]=` not being defined error.